packages/textlint/package.json

```
@@ -30,7 +30,7 @@
     "textlint": "./bin/textlint.js"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.0.0"
```

```suggestion
    "node": ">=6.0.0"
```

Just revert. We will create a BREAKING CHANGE PR.